src/Messaging/WhatsApp/WhatsAppMessageBird.php

```diff
@@ -130,14 +130,15 @@ public function sendTemplateMessage($data)
 
         $content->hsm = $hsmMessage;
 
-        $message = new Message();
-        $message->channelId = $this->whatsAppchannelId;
+        $message = new SendMessage();
         $message->content = $content;
+        $message->from = $this->whatsAppchannelId;
         $message->to = data_get($data, 'to');
         $message->type = 'hsm';
+        $message->reportUrl = data_get($data, 'report_url');
 
         try {
-            return $this->messageBirdClient->conversations->start($message);
+            return $this->messageBirdClient->conversationSend->send($message);
         } catch (Exception $e) {
             Log::error("%s: %s", get_class($e), $e->getMessage());
```

Comment:
```suggestion
            Log::error(printf("%s: %s", get_class($e), $e->getMessage()));
```
you mean printf? bro. otherwise it's error.

Reply:
updated at commit cdbf1d221b60d5f68e2fc256976b40862b9e0710